### PR TITLE
IGAPP-1175: Poi category icons api client

### PR DESCRIPTION
**Pull request by f1sh1918**
Now Poi Category icons are shown on the map in web & native.

Please check:
-  https://github.com/digitalfabrik/integreat-app/pull/871 (Category labels web)
- https://github.com/digitalfabrik/integreat-app/pull/870 (Category labels native)

I would like to merge the categories (icons & labels) together

Check Pois in Testumgebung for labels and icons

### CODE REVIEW

**Review by LeandraH — APPROVED** [review]

**Comment by SteffiStuffel** [comment]
Our naming convention is to use a dash (-) instead of a unterscore (_) to separate words in branch names.

File: native/src/components/__tests__/PoiDetails.spec.tsx [hunk 1]
```diff
@@ -45,6 +45,8 @@ describe('PoiDetails', () => {
     const { getByText } = renderWithTheme(<PoiDetails poi={poi} feature={feature} language={language} />)
 
     expect(getByText(poi.title)).toBeTruthy()
+    // eslint-disable-next-line @typescript-eslint/no-non-null-asserted-optional-chain
+    expect(getByText(poi.category?.name!)).toBeTruthy()
```
Comment (sarahsporck) [inline native/src/components/__tests__/PoiDetails.spec.tsx:49]: shouldn't it then be `poi.category!.name!`? :)

**Review by sarahsporck — APPROVED** [review]
Only tested in web as mobil still strikes. Looks really nice. Great work!

**Comment by f1sh1918** [comment]
> Only tested in web as mobil still strikes. Looks really nice. Great work!

What do you mean with still strikes @sarahsporck the emulator bugs?

**Pull request merged**